proto/table_schedule.proto

```diff
@@ -70,6 +70,7 @@ message Heartbeat {
         (gogoproto.casttype) = "github.com/pingcap/tiflow/cdc/model.TableID",
         (gogoproto.customname) = "TableIDs"
     ];
+    bool is_stopping = 2;
```

Comment:
What about `ask_stop`?

Reply:
`is_stopping` is more simple, and unified.